src/js/DOMUtils.js

```diff
@@ -0,0 +1,91 @@
+// Copyright 2015 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+goog.provide('axs.dom');
+
+/**
+ * Returns the nearest ancestor which is an Element.
```

Comment:
It seems as though `null` is also a possible return type. Should this be documented (e.g., `@return {Element|null}`?